pom.xml

```diff
@@ -95,6 +102,23 @@
     </reporting>
 
     <build>
+        <resources>
```

Comment:
out of curiosity why are we first including and then excluding the same resource